packages/astro-sst/src/adapter.ts

```diff
@@ -24,17 +24,13 @@ function getAdapter({
     exports: ["handler"],
     adapterFeatures: {
       edgeMiddleware: false,
-      functionPerRoute: false,
+      buildOutput: isStatic ? "static" : "server",
```

Comment:
@fwang This is another change I wanted to highlight. Without this, I believe the `buildOutput` would default to `server`. I'm not sure if that's a bug or expected behavior. As a consumer of the API, I'd expect the output to be static if that's what I passed in.